sentry/src/main/java/io/sentry/Hub.java

```diff
@@ -69,7 +69,7 @@ public boolean isEnabled() {
   public @NotNull SentryId captureEvent(
       final @NotNull SentryEvent event, final @Nullable Object hint) {
     SentryId sentryId = SentryId.EMPTY_ID;
-    if (!isEnabled()) {
+    if (!isEnabled) {
```

Comment:
access to the field is always faster than a getter, on Java.
on Dart, for example, field access and getters are the same thing.